platform/ios/src/MGLMapView.mm

```diff
@@ -3525,13 +3525,34 @@ - (MGLCoordinateBounds)convertRect:(CGRect)rect toCoordinateBoundsFromView:(null
     return MGLCoordinateBoundsFromLatLngBounds([self convertRect:rect toLatLngBoundsFromView:view]);
 }
 
+/// Converts MGLCoordinateBounds to a rectangle in the view’s coordinate
+/// system.
 - (CGRect)convertCoordinateBounds:(MGLCoordinateBounds)bounds toRectToView:(nullable UIView *)view
 {
-    if ( ! CLLocationCoordinate2DIsValid(bounds.sw) || ! CLLocationCoordinate2DIsValid(bounds.ne))
-    {
-        return CGRectNull;
-    }
-    return [self convertLatLngBounds:MGLLatLngBoundsFromCoordinateBounds(bounds) toRectToView:view];
```

Comment:
This makes `-convertLatLngBounds:toRectToView:` unused. But instead of deleting that method, consider moving your corrections from `-convertCoordinateBounds:toRectToView:` to `-convertLatLngBounds:toRectToView:`. That way you can perform one less conversion, and `-convertLatLngBounds:toRectToView:` remains available for any conversion needs down the line, robust enough to handle the antimeridian correctly.